go/teams/teams.go

```diff
@@ -850,3 +874,35 @@ func LoadTeamPlusApplicationKeys(ctx context.Context, g *libkb.GlobalContext, id
 	}
 	return team.ExportToTeamPlusApplicationKeys(ctx, keybase1.Time(0), application)
 }
+
+func (t *Team) loadAllTransitiveSubteams(ctx context.Context) ([]*Team, error) {
+	subteams := []*Team{}
+	for subteamID := range t.Data.Chain.SubteamLog {
```

Comment:
Also you're only allowed to do this if the team was loaded with `NeedAdmin`. Is that ok here? Maybe put a note on this `loadAllTransitiveSubteams` method to that effect.

Here's the note on ListSubteam
```
// Only call this on a Team that has been loaded with NeedAdmin.
// Otherwise, you might get incoherent answers due to links that
// were stubbed over the life of the cached object.
func (t *TeamSigChainState) ListSubteams() (res []TeamIDAndName) {
```